tools/cachestat.py

```diff
@@ -139,24 +139,16 @@ def usage():
     for k, v in sorted(counts.items(), key=lambda counts: counts[1].value):
 
         if re.match('mark_page_accessed', b.ksym(k.ip)) is not None:
-            mpa = v.value
-            if mpa < 0:
-                mpa = 0
+            mpa = max(0, mpa)
```

Comment:
/facepalm. totally.... I only have 1 line right which was enough to visually look sane :s